src/importIssues.ts

```diff
@@ -70,6 +76,36 @@ export const importIssues = async (apiKey: string, importer: Importer) => {
         return !answers.newTeam;
       },
     },
+    {
+      type: 'confirm',
+      name: 'includeProject',
+      message: 'Do you want to import to a specific project?',
+      when: answers => {
+        return !answers.newTeam;
```

Comment:
Maybe another check for if their Team has projects?

Reply:
Good call. Added check for projects before continuing to that flow